theme/static/js/generic-resource.js

```diff
@@ -91,11 +91,30 @@ function addEditExtraMeta2Table() {
     // Restore validation UI state
     $("#extra_meta_msg").hide();
     $("#extra_meta_name_input").removeClass("form-invalid");
+    $("#extra_meta_value_input").removeClass("form-invalid");
     var t = $('#extraMetaTable').DataTable();
+
     var extra_meta_name = $("#extra_meta_name_input").val().trim();
     var extra_meta_value = $("#extra_meta_value_input").val().trim();
     var edit_extra_meta_row_id = $("#edit_extra_meta_row_id").val().trim();
 
+    var sanitized_extra_meta_name = $("<div/>").html(extra_meta_name.trim()).text();
```

Comment:
Would be better to use a helper function to check if a string contains HTML tags.  Since it's going to be a tiny function will leave up to you to make the change.

Reply:
done